experiments:

  - name: tabletop_01
    goal: below_table_01
    pre_action: nothing
    post_action: nothing
    sound_bite: "testing is fun!"

